apps/fishing-map/features/reports/vessel-groups/insights/vessel-group-report-insights.selectors.ts

```diff
@@ -36,7 +36,8 @@ export const selectVGRVesselsByInsight = <Insight = any>(
     if (!data || !vesselGroup) {
       return []
     }
-    const insightVessels = vesselGroup?.vessels?.flatMap((vessel) => {
+    const vesselsWithoutDuplicates = vesselGroup?.vessels.filter((v) => v.identity !== undefined)
```

Comment:
This might be moved to a util since is also used in `apps/fishing-map/features/reports/events/vgr-events.selectors.ts`